src/test/js/shared/idam_helper.js

```diff
@@ -241,6 +241,33 @@ class IdamHelper extends Helper {
          browser.close();
        });
    }
+
+  extractUrl(email) {
+     return (notifyClient
+         .getNotifications("email", "sending")
+         .then(response => {
+             console.log("Searching " + response.body.notifications.length + " emails(s)");
+             var result = response.body.notifications.find(obj => {
+                 if (obj.email_address === email) {
+                     return obj.email_address === email
+                 } else {
+                     console.log("ignoring unmatched email address " + obj.email_address);
```

Comment:
best comment this out - could potentially put real email addresses in the log

Reply:
Yep, was debug

Reply:
Actually, can you remove the whole else block and put in a comment that says "NOTE:  NEVER LOG EMAIL ADDRESSES FROM THE PRODUCTION QUEUE". Once we create the other notify keys it won't be an issue, but for now...